### ISSUE TITLE
String concatenation with System.get_env constants lints false negative

### ISSUE THREAD

**Issue by holyxiaoxin** [AtomLinter/linter-elixirc#43]
```
@api_uri "http://some_api.com"
@token System.get_env("SOME_ENV")

def some_function do
    uri = @api_uri <> @token
end
```

The line `uri = @api_uri <> @token` will produce a false lint error/message.
-> `invalid literal nil in <<>>`

**Comment by Arcanemagus** [comment]
Does running `elixirc` from a terminal show the same issues? If so then this is an upstream bug.

**Comment by Arcanemagus** [comment]
Marking this as closed due to a lack of response, if you are still having issues please provide the requested information.

**Comment by Arcanemagus** [comment]
As a note, I'm not seeing any message from this code with Elixir 1.4.1, so whatever the issue was it seems to have been fixed on their end in the time since this was filed.
